core/src/main/java/org/apache/cxf/common/util/SpringClasspathScanner.java

```diff
@@ -170,7 +170,8 @@ private static boolean isSpringInOsgi() {
 
     private ResourcePatternResolver getResolver(ClassLoader loader) {
         ResourcePatternResolver resolver = null;
-        if (IN_OSGI) {
+        inOSGI = isSpringInOsgi();
```

Comment:
move this call as initial initialization to avoid 'isSpringInOsgi' call for each 'getResolver' invoke

Reply:
@amarkevich  The best place is to revert and place a static initializer  since  it is not good idea to put a static method call into the constructor

Reply:
expected impl:
`private static final boolean IN_OSGI = isSpringInOsgi();`

Reply:
@amarkevich  done with thanks.